python/imjoy_rpc/rpc.py

```diff
@@ -68,8 +71,9 @@ def __init__(self, connection, rpc_context, config=None, codecs=None):
         super().__init__(self._remote_logger)
 
         try:
+            # FIXME: What exception do we expect?
             self.loop = asyncio.get_event_loop()
-        except:
+        except Exception:
```

Comment:
Should be a RuntimeError

https://techoverflow.net/2020/10/01/how-to-fix-python-asyncio-runtimeerror-there-is-no-current-event-loop-in-thread/